### PR TITLE
Add mult estimators to matching debug output

### PR DESCRIPTION
**Pull request by shahor02**

### CODE REVIEW

File: Detectors/GlobalTracking/src/MatchTPCITS.cxx [hunk 1]
```diff
@@ -44,7 +44,9 @@
 #include "DetectorsBase/GRPGeomHelper.h"
 #include "ITStracking/IOUtils.h"
 
-#include "GPUO2Interface.h" // Needed for propper settings in GPUParam.h
+// #include "GPUO2Interface.h" // Needed for propper settings in GPUParam.h
```
Comment (davidrohr) [inline Detectors/GlobalTracking/src/MatchTPCITS.cxx]: Why did you remove this header? Does it create any problems? In principle, it is still needed to get the correct defines for GPUParam, etc.

Comment (shahor02) [reply Detectors/GlobalTracking/src/MatchTPCITS.cxx]: Actually, I've commented it out when was fighting with compilation (missing param.inc) and then forgot to uncomment. But it works w/o it.

**Comment by davidrohr** [comment]
Probably works without since it is pulled in from somewhere else before.

**Comment by shahor02** [comment]
CIs were green, just restorerd one include, merging
